puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-pipeline.groovy

```diff
@@ -67,6 +67,7 @@ pipeline {
                 // TODO: from variables
                 parallel(
                     "stretch": { push_debs_direct('stretch', foreman_version) },
+                    "buster": { push_debs_direct('buster', foreman_version) },
```

Comment:
This means it'll also attempt to push for buster on existing releases. Hence the TODO to get it from variables :)

Reply:
That would mean, it would get pushed for e.g. 1.23?

Reply:
deleted.

Reply:
Correct. We'll need to do some groovy magic to make that configurable.